framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -561,43 +555,26 @@ private final void printPaths() {
      *
      * @param annoName canonical name of an external annotation class, e.g.
      *     "myproject.qual.myannotation"
-     * @return the loaded annotation class
+     * @return the loaded annotation class, or null if it was not a supported annotation as decided
```

Comment:
Decided how?

Reply:
Updated comment to point to `isSupportedAnnotationClass()`